tests/api/onboarding-tasks.php

```diff
@@ -30,6 +30,16 @@ public function setUp() {
 				'role' => 'administrator',
 			)
 		);
+
+		$products = wc_get_products(
+			array(
+				'return' => 'ids',
+			)
+		);
+		// delete all products.
```

Comment:
I feel like this kind of thing probably belongs more in a `tearDown` than `setUp`.  Are these products coming from somewhere else or are there defaults added prior to running any tests?

Reply:
I believe coming from somewhere else, as I check if at-least more then 10 are created in the product import test. If products with the same SKU already exists it skips them.

Reply:
I moved it to the tearDown, although I do call delete products at the start of the failing test as well as tearDown only gets called after the fact - https://github.com/woocommerce/woocommerce-admin/pull/6178/commits/7991abaf0cab21db177a6da9280c1916cd7e490b

Reply:
I think I might have been unclear.  What I meant is:

Are there other tests that create products?  If so, product deletion should go into that test's tear down.  I don't think its an individual test's responsibility to reset the environment and creates a bad habit for us to not clean up after we're done testing something.

If products exist by default on all tests, then I think leaving it in the setup is okay.

That said, if this causes a lot of headaches or is difficult to track down, we can leave it in the setup and tackle stabilizing the environment in a follow-up.

Reply:
Gotcha, yeah I could spent sometime during the cooldown to have a look at that. This PR was mostly a quick fix to get the tests up and working again.
I actually removed the `delete all products` for now, and changed the deleting attribute to only color and logo, which both of them rely on. Although `unregister_taxonomy` isn't the right way to do this I have noticed.

I can certainly re-think this, but didn't want to hold up all the other PRs, would skipping them for now be better?